sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java

```diff
@@ -53,6 +56,8 @@ public IdentityClientOptions() {
         retryTimeout = i -> Duration.ofSeconds((long) Math.pow(2, i.getSeconds() - 1));
         regionalAuthority = RegionalAuthority.fromString(
             configuration.get(Configuration.PROPERTY_AZURE_REGIONAL_AUTHORITY_NAME));
+        identityLegacyTenantSelection = configuration
+            .get(AZURE_IDENTITY_ENABLE_LEGACY_TENANT_SELECTION, false) ;
```

Comment:
Is this env var set by the user? If so, should we make this constant publicly available? It is public in this class but it's in implementation package.